docker/docker_official_images/3.8.0/jvm/Dockerfile

```diff
@@ -92,4 +87,4 @@ USER appuser
 
 VOLUME ["/etc/kafka/secrets", "/var/lib/kafka/data", "/mnt/shared/config"]
```

Comment:
Are _all_ of these paths intended to be user-specified / persistent storage?  These are _all_ spearately paths that Kafka will write to at runtime in the default configuration and that users will have a bad time if they don't save in some persistent place/way?